modules/apps/sample-portlet/poshi-runner.properties

```diff
@@ -0,0 +1,4 @@
+test.name=SamplePortlet#smoke
+test.console.log.file.name=${project}/tomcat-8.0.32/logs
```

Comment:
Could this somehow be configured by workspace plugin?  Because this requires hard-coding the tomcat path which it could change based on the portal bundle version.